Source/LocationInfo.swift

```diff
@@ -0,0 +1,21 @@
+public class LocationInfo {
+    //    #namespace     String   The name of the namespace / module
+    //    #file          String   The name of the file in which it appears.
+    //    #line          Int      The line number on which it appears.
+    //    #column        Int      The column number in which it begins.
+    //    #function      String   The name of the declaration in which it appears.
+    //    #dsohandle     String   The dso handle.
+    public var file : String?
+    public var line : Int?
+    public var column : Int?
+    public var function : String?
+
+    init (file: String? = nil, line: Int? = nil, column: Int? = nil, function: String? = nil) {
```

Comment:
As I noted in https://github.com/Zewo/Log/pull/2/files#r58325211 - it might be a good idea to replace the `file: String? = nil` with `file: String = #file`.